casepro/statistics/signals.py

```diff
@@ -52,3 +53,30 @@ def record_incoming_labelling(sender, instance, action, reverse, model, pk_set,
     elif action == 'pre_clear':
         for label in instance.labels.all():
             DailyCount.record_removal(day, DailyCount.TYPE_INCOMING, label)
+
+
+@receiver(post_save, sender=Case)
+def record_new_case(sender, instance, created, **kwargs):
+    """
+    This is where we keep track of DailyCounts for partners
+    """
+    day = datetime_to_date(instance.opened_on, instance.org)
+    if instance.closed_on:
+        DailyCount.record_item(day, DailyCount.TYPE_CASE_CLOSED, instance.assignee)
+    else:
+        DailyCount.record_item(day, DailyCount.TYPE_CASE_OPENED, instance.assignee)
```

Comment:
Don't think you need these if also tracking CaseAction's below? Tracking CaseActions is preferable because these will record any update to a Case instance as an open or close.